index.html

```diff
@@ -498,9 +501,57 @@
       <div class="division-stripe">
       </div>
 
-      <section class="location container">
-            <h2>Localização</h2>
-            <div id="map"></div>
+      <section class="faq container">
+        <h2>Perguntas frequentes</h2>
+        <ul>
+          <li>
+            <input type="checkbox" checked>
+            <h3>Como funcionam as inscrições?</h3>
+            <p>
+              Clica lá se inscreve se der sorte deu bom
+            </p>
+          </li>
+          <li>
+            <input type="checkbox" checked>
+
+            <h3>Posso levar meu Notebook?</h3>
+            <p>
+              Pode e deve
+            </p>
+          </li>
+          <li>
+            <input type="checkbox" checked>
+
+            <h3>Vou ganhar certificado?</h3>
+            <p>
+              Sim
```

Comment:
We need to increase this answer, normally just a person that gives his name and check the attendance list
gets the certification